src/api/hooks/video.ts

```diff
@@ -52,3 +56,44 @@ export const useAddVideoView = (opts?: AddVideoViewOpts) => {
     ...rest,
   }
 }
+
+type MostViewedVideosOpts = QueryHookOptions<GetMostViewedVideosQuery>
+export const useMostViewedVideosIds = (variables?: GetMostViewedVideosQueryVariables, opts?: MostViewedVideosOpts) => {
+  const { data, ...rest } = useGetMostViewedVideosQuery({ ...opts, variables })
+  return {
+    mostViewedVideos: data?.mostViewedVideos,
+    ...rest,
+  }
+}
+
+export const useMostViewedVideos = (variables?: GetMostViewedVideosQueryVariables, opts?: MostViewedVideosOpts) => {
+  const { mostViewedVideos } = useMostViewedVideosIds(variables, opts)
+
+  const mostViewedVideosIds = useMemo(() => {
+    if (mostViewedVideos) {
+      return mostViewedVideos.map((item) => item.id)
+    }
+    return null
+  }, [mostViewedVideos])
+
+  const { videos, ...rest } = useVideos(
+    {
+      where: {
+        id_in: mostViewedVideosIds,
+      },
+    },
+    { skip: !mostViewedVideosIds }
+  )
+
+  const sortedVideos = useMemo(() => {
+    if (videos) {
+      return [...videos].sort((a, b) => (b.views && a.views ? b.views - a.views : 0))
+    }
+    return null
+  }, [videos])
```

Comment:
it should sort the videos by views, right? 
But I dont see them sorted on front page: 
![image](https://user-images.githubusercontent.com/51168865/127480703-7e4b80a8-f62c-48ec-ac31-0189144c7010.png)

Reply:
Okay, everything is clear but will leave this comment though, because Rafał gave me a small explanation in pm. 
This hook is not used at all right now, but will be needed in the future(for carousel). For now, only `useMostViewedVideosIds` is used in the Homeview.

Reply:
Can I ask why is not used?